Makefile

```diff
@@ -163,9 +163,9 @@ DEFENTROPYSOURCE := /dev/urandom
 DEFDISABLEBLOCK := false
 DEFSHAREDFS := virtio-9p
 DEFSHAREDFS_NEMU := virtio-fs
-DEFVIRTIOFSDAEMON := $(VIRTIOFSDBINDIR)/virtiofsd
+DEFVIRTIOFSDAEMON := $(VIRTIOFSDBINDIR)/virtiofsd-x86_64
 # Default DAX mapping cache size in MiB
-DEFVIRTIOFSCACHESIZE := 8192
+DEFVIRTIOFSCACHESIZE := 1024
```

Comment:
Some hint of why the size change would be nice ;-)

Reply:
done (in commit message)